src/components/local-scene-sidebar/overview-sidebar/local-priority-card/local-priority-card.js

```diff
@@ -1,23 +1,37 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { connect } from 'react-redux';
 import metadataActions from 'redux_modules/metadata';
-import metadataConfig from 'constants/metadata';
 import Component from './local-priority-card-component';
 
+import metadataConfig, {
+  MERGED_PROTECTION,
+  COUNTRY_PRIORITY
+} from 'constants/metadata';
+import metadataService from 'services/metadata-service';
+
 const LocalPriorityCardContainer = props => {
-  const handleInfoClick = slug => {
-    const { setModalMetadata } = props;
-    const md = metadataConfig[slug];
-    setModalMetadata({
-      slug: md.slug,
-      title: md.title,
-      isOpen: true
-    });
-  }
+
+  const [priorityMetadata, setPriorityMetadata] = useState(null);
+  const [protectionMetadata, setProtectionsMetadata] = useState(null);
+
+  useEffect(() => {
+    const md = metadataConfig[MERGED_PROTECTION]
+    metadataService.getMetadata(md.slug).then( data => {
+      setProtectionsMetadata(data);
+    })
+  }, []);
+
+  useEffect(() => {
+    const md = metadataConfig[COUNTRY_PRIORITY]
+    metadataService.getMetadata(md.slug).then( data => {
+      setPriorityMetadata(data);
+    })
+  }, []);
```

Comment:
Just wondering, why not group them on the same useEffect as they are going to happen on mount?

Reply:
Just a matter of taste. Having each `effect` on its own function looks cleaner to me 😅